Aztec/Classes/NSAttributedString/HTML/HTMLRepresentation.swift

```diff
@@ -0,0 +1,4 @@
+import Foundation
+
+protocol HTMLRepresentation {
+}
```

Comment:
Regarding if it's necessary - it will be.  Otherwise you can't have properties or arrays that can contain both `HTMLElementRepresentation` and `HTMLAttributeRepresentation` objects.

Since a style can be represented by either an element or an attribute, unless there's a better alternative we can implement right now, I'd still go forward with this.

Regarding moving the protocol, I'm not sure how that makes sense since the protocol is not related to `HTMLElementRepresentator` in any way, other than the latter implementing the protocol.